### PR TITLE
Updated .env.example (CU-86dun7tc2)

### PR DESCRIPTION
**Pull request by Nakul727**

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-86dun7tc2 Update the .env.example for Sensors](https://app.clickup.com/t/86dun7tc2)

**Review by Darrickg — APPROVED** [review]
LGTM! wait for Johnny and Corey's approval though

**Review by CCheng00 — APPROVED** [review]
looks good!

**Pull request merged**
